src/main/java/org/primefaces/util/DynamicContentSrcBuilder.java

```diff
@@ -65,8 +65,12 @@ else if (value instanceof String) {
         }
         else if (value instanceof StreamedContent) {
             StreamedContent streamedContent = (StreamedContent) value;
+            ValueExpression ve = null;
+            if (attributeName != null && attributeName.length() > 0) {
```

Comment:
Could this be `!LangUtils.isValueBlank(attributeName)` ?

Reply:
Done. I always search for StringUtils and can´t remember LangUtils...